packages/app/src/specs/banners/TrackedBanner.vue

```diff
@@ -10,16 +10,26 @@
 
 <script setup lang="ts">
 import Alert from '@packages/frontend-shared/src/components/Alert.vue'
-import { watchEffect } from 'vue'
+import { ref, watchEffect } from 'vue'
 import { gql, useMutation, useQuery } from '@urql/vue'
-import { TrackedBanner_ProjectStateDocument, TrackedBanner_SetProjectStateDocument } from '../../generated/graphql'
+import { TrackedBanner_ProjectStateDocument, TrackedBanner_RecordBannerSeenDocument, TrackedBanner_SetProjectStateDocument } from '../../generated/graphql'
 import { set } from 'lodash'
+import { useDebounceFn } from '@vueuse/core'
+import { nanoid } from 'nanoid'
```

Comment:
I don't see `nanoid` in any of our `package.json`s, can we add it to the app? We shouldn't rely on a transitive dep.